bee-api/bee-rest-api/src/endpoints/config.rs

```diff
@@ -220,7 +215,18 @@ impl RestApiConfig {
     }
 
     /// Returns the white flag solidification timeout.
-    pub fn white_flag_solidification_timeout(&self) -> u64 {
+    pub fn white_flag_solidification_timeout(&self) -> Duration {
         self.white_flag_solidification_timeout
     }
 }
+
+pub(crate) fn route_to_regex(route: &str) -> String {
+    // Escape the string to make sure a regex can be built from it.
+    // Existing wildcards `*` get escaped to `\\*`.
```

Comment:
`*` actually is not a meta character in regex, so it is not escaped. See [`regex_syntax`](https://docs.rs/regex-syntax/latest/src/regex_syntax/lib.rs.html#178-182) which is called by [`regex`](https://docs.rs/regex/latest/src/regex/re_unicode.rs.html#21-23).

Also, `regex::escape` does not consider unicode characters (\w, \d, \s), but you are not using the feature for those so it will error if someone did try to use them. 🤷